.changeset/curly-jobs-sort.md

```diff
@@ -0,0 +1,6 @@
+---
+"@skeletonlabs/skeleton-svelte": minor
+---
+
+bugfix(avatar): removed unecessary h-full class from fallbackbase. 
```

Comment:
Go ahead and drop the `(avatar)` portion in the prefix. That should only be `bugfix:`. This appears verbatim on the changelog so something public users will see.

Reply:
thank you Chris. Im still getting the patterns and styles of the project. I expect to contribute much more! Thank you for the atention, i will change the pr now

Reply:
I've made the change Chris, I hope I've done right this time. I appologize for my errors and my lack of experience in open source, but i intend to be a valuabe contributor to the project